ext/opentelemetry-ext-wsgi/src/opentelemetry/ext/wsgi/flask_util.py

```diff
@@ -0,0 +1,88 @@
+# Note: This package is not named "flask" because of
+# https://github.com/PyCQA/pylint/issues/2648
+
+import logging
+
+from flask import request as flask_request
+
+import opentelemetry.ext.wsgi as otel_wsgi
+from opentelemetry import propagators, trace
+from opentelemetry.util import time_ns
+
+logger = logging.getLogger(__name__)
+
+_ENVIRON_STARTTIME_KEY = object()
+_ENVIRON_SPAN_KEY = object()
+_ENVIRON_ACTIVATION_KEY = object()
+
+
+def wrap_flask(flask):
```

Comment:
isn't the "flask" here redundant since it's it's in the flask_util module?

I'd say it's more intuitive to name this "wrap_app"

Reply:
I renamed it to `instrument_app`.